backend/tests/sessions/hooks/test_provisioning_hook.py

```diff
@@ -158,3 +159,37 @@ def test_tool_model_mismatch(
                 )
             ],
         )
+
+
+def test_provision_session_with_compatible_tool_versions(
+    db: orm.Session,
+    admin: users_models.DatabaseUser,
+    tool_version: tools_models.DatabaseVersion,
+    capella_tool: tools_models.DatabaseTool,
+    capella_tool_version: tools_models.DatabaseVersion,
+    project: projects_models.DatabaseProject,
+    capella_model: toolmodels_models.DatabaseToolModel,
+    git_model: git_models.DatabaseGitModel,
+):
+    """Make sure that provisioning is successful when the tool is compatible with the tool of the model"""
+
+    tool_version.config.compatible_versions = [capella_tool_version.id]
+    orm.attributes.flag_modified(tool_version, "config")
+    db.commit()
+
+    response = hooks_provisioning.ProvisionWorkspaceHook().configuration_hook(
+        db=db,
+        tool=capella_tool,
+        tool_version=capella_tool_version,
```

Comment:
```suggestion
        tool=tool,
        tool_version=tool_version,
```
This is needed because we want to test if we can request a session with tool `tool` and tool version `tool_version` using a Capella model.

Reply:
Good catch, I've changed it.